zerver/lib/message.py

```diff
@@ -168,7 +168,10 @@ class SendMessageRequest:
     default_bot_user_ids: Set[int]
     service_bot_tuples: List[Tuple[int, int]]
     all_bot_user_ids: Set[int]
+    topic_wildcard_mention_user_ids: Set[int]
     stream_wildcard_mention_user_ids: Set[int]
+    # IDs of users who have followed the topic the message (having topic wildcard) is being sent to, and have the followed topic wildcard mentions notify setting ON.
+    topic_wildcard_mention_in_followed_topic_user_ids: Set[int]
     # IDs of users who have followed the topic the message (having stream wildcard) is being sent to, and have the followed topic wildcard mentions notify setting ON.
```

Comment:
These comments should be line wrapped to 80 characters.

Reply:
Done